# 8 * uint16 for IR sensors
uint16 ch1
uint16 ch2
uint16 ch3
uint16 ch4
uint16 ch5
uint16 ch6
uint16 ch7
uint16 ch8

# Say if battery is in use, of if a wall adaptator is present
# value : 1 if on battery, 0 if adaptator present
uint8 on_batt

# cell voltage
float32 cell1
float32 cell2
float32 cell3



